python/ynab_client.py

```diff
@@ -65,6 +67,3 @@ def findPreviousTransaction(payee_name):
         .filter(Transaction.imported_payee==payee_name)\
         .order_by(Transaction.date.desc())\
         .first()
-
-# Sync with YNAB
-sync()
```

Comment:
Yeah, it's tricky. Just for some context, Heroku has a 30-second timeout for web requests and a 60-second timeout for process booting. So we moved it there as it works a bit better for bigger accounts.